src/Pages/ShopPage/ShopComponents/Sidenav/Sidenav.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import "./Sidenav.css";
+import { useFilter } from '../../../../Contexts/filter-controller'
+
+const Sidenav = () => {
+
+    const { dispatch, filteredState } = useFilter()
+
+    const handleClearFilters = () => dispatch({ type: "CLEAR_ALL", payload: { filterKey: true, value: true } })
```

Comment:
fix: On clicking the clear All button the filter are not getting cleared

Reply:
for some reason the function only works when its given inline. fixed.